packages/loader/container-loader/README.md

```diff
@@ -79,7 +83,19 @@ When container is closed, the following is true (in no particular order):
 3. "readonly" event fires on DeltaManager & Container (and Container.readonly property is set to true)  indicating to all data stores that container is read-only, and data stores should not allow local edits, as they are not going to make it.
 4. "disconnected" event fires, if connection was active at the moment of container closure.
 
-`"closed"` event is available on Container for hosts. `"disposed"` event is delivered to container runtime when container is closed. But container runtime can be also disposed when new code proposal is made and new version of the code (and container runtime) is loaded in accordance with it.
+`"closed"` event is available on Container for hosts.
+
+#### `Container.dispose()`
```

Comment:
I do not think current description clearly answers these two questions:
- When should developers call dispose() vs. close()?
- all the places where they are different (you did not mention the main one that created this task - raising "readonly" events).

I'd also try not to duplicate info. I.e. I'd change your list below to say - "it works exactly like close(), but here is a list of differences". I.e. list should not talk about "closed" event or "disconnected" events, it should only say that "disposed" event is raised, but "readonly " events are not.